data/schema/SSVC_Computed_v2.01.schema.json

```diff
@@ -9,15 +9,20 @@
 	    "type": "array",
 	    "items": {
 		"type": "object",
-		"items": {
-		    "type": "string"
+		"patternProperties": {
+		    ".*": {
+			"type": ["array","string"],
+			"items": {
+			    "type": "string"
+			}
+		    }
 		}
 	    },
 	    "minItems": 1,
 	    "uniqueItems": true		
 	},
 	"computed": {
-	    "description": "Computed score short representation such as SSVCv2/Ps:Nm/T:T/U:E/1605040000/ for a vulnerability with no or minor Public Safety Impact,  total Technical Impact, and efficient Utility, which was evaluated on Nov 10, 2020.",
+	    "description": "Computed score short representation such as SSVC/v:2/R:C/E:[P,A]/V:R/T:P/M:H/D:[T,R,A,E]/1607626684/ for a vulnerability with no or minor Public Safety Impact,  total Technical Impact, and efficient Utility, which was evaluated on Nov 10, 2020.",
```

Comment:
looks like you have version and role split out here, but not in https://github.com/CERTCC/SSVC/pull/107/files#diff-59989ecd9b70fdbb329640c6da907e5fcc247ba37bc20eaeac267f325e7f70e0 so this might just be a cleanup for consistency before merging this PR.

Reply:
All changes look right.  One thing I want to be clear is v 2.01 mentioned is about the JSON file, it is NOT a new version of SSVC.  Sorry I should picked a different number scheme perhaps. Hope that makes sense.

It will be worth in next two weeks or so for three of us to meet and sort of clarify what our thoughts about things to cover and close in the next few weeks.